controllers/storagecluster/cephfilesystem.go

```diff
@@ -42,6 +42,7 @@ func (r *StorageClusterReconciler) newCephFilesystemInstances(initData *ocsv1.St
 				DataPools: []cephv1.NamedPoolSpec{
 					{
 						PoolSpec: cephv1.PoolSpec{
+							DeviceClass:   "replicated",
```

Comment:
please note this should not cause any issue for standalone upgraded clusters and managed service upgraded cluster and also for new managed service feature example https://github.com/red-hat-storage/ocs-operator/pull/1750
cc @nb-ohad

Reply:
ack, Will take a look